src/views/News/components/NewsItem/index.jsx

```diff
@@ -167,10 +237,12 @@ NewsItem.propTypes = {
     // Expiration: PropTypes.string.isRequired,
   }).isRequired,
 
-  unapproved: PropTypes.any,
+  isUnapproved: PropTypes.any,
   size: PropTypes.string.isRequired,
   handleNewsItemEdit: PropTypes.func.isRequired,
   handleNewsItemDelete: PropTypes.func.isRequired,
+  handleNewsApprovalStatus: PropTypes.func.isRequired,
```

Comment:
```suggestion
  handleChangeNewsApprovalStatus: PropTypes.func.isRequired,
```
In naming handlers such as this, we want to handle an "event". That means that the function should probably be named after some kind of event like a change.